app/controllers/claims_form_callbacks.rb

```diff
@@ -73,6 +73,11 @@ def check_your_answers_after_form_save_success
     create_and_save_claim_form
   end
 
+  def employee_email_after_form_save_failure
+    session[:slugs].delete("employee-email")
+    render_template_for_current_slug
+  end
```

Comment:
Are we not just missing a validation somewhere?
If the `practitioner_email_address` is blank, than the claim should not be valid, and the ClaimSubmissionForm should not be valid.
I can't see how to add this validation to just the EY Authenticated journey - so actually, this callback may be the best solution.

Reply:
Either way, I think this needs a feature spec in `spec/features/early_years_payment/provider/authenticated`

Reply:
👍🏽 happy with it as it is for now.
My worry about a lack of feature spec, is when we refactor, we re-introduce this bug.